packages/client/hmi-client/src/router/index.ts

```diff
@@ -15,7 +15,7 @@ import { RouteName } from './routes';
 
 export enum RoutePath {
 	Home = '/',
-	Document = '/docs/:id?',
+	Document = '/projects/:projectId/docs/:id?',
```

Comment:
We might want to figure something about that. Because we want to be able to share documents without having to open a project, maybe we need a separate page just for that purposes. Anyway good for now.

Reply:
Agreed.